docs/src/Modal.doc.js

```diff
@@ -23,85 +34,492 @@ card(
         type: 'string',
         required: true,
         description:
-          'String that clients such as VoiceOver will read to describe the modal. Always localize the label.',
-        href: 'accessibility',
+          'String that clients such as VoiceOver will read to describe the modal. Always localize the label. See the [Accessibility section](#Accessibility) for more info.',
       },
       {
         name: 'align',
         type: `"center" | "left"`,
```

Comment:
This is now out of date with @ohariz 's latest PR, so don't forget to rebase. You may want to update the copy.